index.html

```diff
@@ -45,34 +45,33 @@
               <a class="navigation__link" href="#">Contact Us</a>
             </li>
           </ul>
-        </div>
+        </nav>
 
         <button>Donate Now</button>
 
-    </div>
+      </header>
 
     <div class="content">
-        <div class="main">
-          <div class="alert">
+        <main class="main">
+          <p class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
-          </div>
-
+          </p>
           <div class="hero">
             <h1>Bikes for Refugees</h1>
             <p>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
             <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="volunteer-btn">Volunteer</button>
           </div>
 
-          <div>
+          <div class="container-article-main">
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
-              <div class="article">
+            <div class="article-main">
+              <article class="article article-main-content">
```

Comment:
Watch out when you use multiple classes on one element (e.g. `article` and `article-main-content`) in case you duplicate styles in these two classes (e.g. both of these classes set `display: flex`, but you only need to set this property once)